store-implementation/federated-store/src/test/java/uk/gov/gchq/gaffer/federatedstore/FederatedStoreAuthTest.java

```diff
@@ -150,7 +150,7 @@ public void shouldNotShowHiddenGraphsInError() throws Exception {
                     federatedStore);
             fail("exception expected");
         } catch (final OperationException e) {
-            assertEquals(String.format("Error adding graph %s to storage due to: User is attempting to overwrite a graph within FederatedStore. GraphId: %s", EXPECTED_GRAPH_ID, EXPECTED_GRAPH_ID), e.getCause().getMessage());
+            assertTrue(e.getCause().getMessage().contains(String.format("User is attempting to overwrite a graph within FederatedStore. GraphId: %s", EXPECTED_GRAPH_ID, EXPECTED_GRAPH_ID)));
```

Comment:
If you want you can make way nicer exception tests with assertJ, e.g. https://github.com/gchq/Gaffer/blob/0895c30ef15665e0eff0aef6fe8885bb4877f0d2/core/cache/src/test/java/uk/gov/gchq/gaffer/cache/CacheTest.java#L71-L73